.travis.yml

```diff
@@ -16,6 +17,18 @@ branches:
   - master
   - /^v\d+\.\d+\.\d+/
 
+before_install:
+- |
+    if [[ "$TRAVIS_OS_NAME" = "windows" ]]; then
+      choco install vcbuildtools
+      export NPM_CONFIG_PREFIX=c:\\custom_npm PATH="/c/custom_npm:$PATH" VCTargetsPath="C:\\Program Files (x86)\\MSBuild\\Microsoft.cpp\\v4.0\\v140"
+      npm install -g npm@6
```

Comment:
A recent-ish version of NPM is needed to bring in `node-gyp@3.8.0`, so it knows what VS2015 is.